test/instrumentatios/static-instrumentations/grpc-js/generated_proto/hello_grpc_pb.d.ts

```diff
@@ -0,0 +1,90 @@
+// package: helloworld
+// file: hello.proto
+
+/* tslint:disable */
+/* eslint-disable */
+
+import * as grpc from '@grpc/grpc-js';
```

Comment:
a potentially big import, maybe limit it to relevant methods

Reply:
That was protobuf generated